CHANGELOG.md

```diff
@@ -38,6 +38,8 @@
 
 - Enable `expressions` and `traditional_compatible` router flavor in stream subsystem.
   [#11071](https://github.com/Kong/kong/pull/11071)
+- Make upstream `host_header` and router `preserve_host` config work in stream tls proxy.
```

Comment:
The naming "`host_header`" is somehow misleading. But introducing a new field is also risky. How about making an aliasing?

Reply:
How to make an aliasing?

Reply:
I think we do not need to do this, let these two field in the stream proxy seems Ok, and do not need some downstream product change anything.(Konnect or other)